main.cpp

```diff
@@ -92,7 +92,128 @@ good to go!
  */
 
 #include <iostream>
-int main() 
+
+class FloatType
+{
+public:
+    FloatType()
```

Comment:
Please use `struct`.  it's public by default
Save yourself some typing
- [ ] revise